bin/hbase

```diff
@@ -480,6 +480,11 @@ add_jdk11_deps_to_classpath() {
   done
 }
 
+enable_trace() {
+  agent_jar=$(find lib/trace -type f -name "opentelemetry-javaagent-*")
+  HBASE_OPTS="$HBASE_OPTS -javaagent:$agent_jar $HBASE_TRACE_OPTS"
```

Comment:
It'll just fail if two classes w/ same prefix... that is fine I think and shouldn't happen usually.